test_install/CMakeLists.txt

```diff
@@ -37,10 +37,9 @@ if(GINKGO_BUILD_CUDA)
     target_link_libraries(test_install_cuda PRIVATE Ginkgo::ginkgo)
 endif()
 
-if(GINKGO_BUILD_HIP
-   AND GINKGO_HIP_PLATFORM MATCHES "hcc"
-   AND GINKGO_HIP_VERSION VERSION_GREATER_EQUAL "3.5"
-   AND NOT GINKGO_BUILD_SHARED_LIBS)
+if(GINKGO_BUILD_HIP AND GINKGO_HIP_PLATFORM MATCHES "hcc"
+        AND GINKGO_HIP_VERSION VERSION_GREATER_EQUAL "3.5"
+        AND NOT GINKGO_BUILD_SHARED_LIBS)
```

Comment:
different format?

Reply:
The format is supposed to be 4 spaces per block. Somehow this counts as two blocks on my setup (one the if, another the parenthesis?).